<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolSettingsDialog</class>
 <widget class="QDialog" name="ToolSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tool Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="settingTabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pathSettingsTab">
      <attribute name="title">
       <string>Paths</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="functionLibraryLabel">
         <property name="text">
          <string>Function Library</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="functionLibraryLayout">
         <item>
          <widget class="QLineEdit" name="functionLibraryLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="functionLibraryBrowseButton">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="platformGeneratorLabel">
         <property name="text">
          <string>Platform Generator</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="platformGeneratorPathLayout">
         <item>
          <widget class="QLineEdit" name="platformGeneratorLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="platformGeneratorBrowseButton">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="simulatorLabel">
         <property name="text">
          <string>Simulator</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="simulatorPathLayout">
         <item>
          <widget class="QLineEdit" name="simulatorLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="simulatorBrowseButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>30</horstretch>
             <verstretch>25</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="builderLabel">
         <property name="text">
          <string>Builder</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="builderPathLayout">
         <item>
          <widget class="QLineEdit" name="builderLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="builderBrowseButton">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="compilerSettingsTab">
      <attribute name="title">
       <string>Compilers</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ToolSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ToolSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
